lib/fog/openstack/requests/compute/create_server.rb

```diff
@@ -169,6 +169,12 @@ def create_server(name, image_ref, flavor_ref, options = {})
 
           self.data[:last_modified][:servers][server_id] = Time.now
           self.data[:servers][server_id] = mock_data
+
+          if options['os:scheduler_hints'] && options['os:scheduler_hints']['group']
+            grp = self.data[:server_groups][options['os:scheduler_hints']['group']]
```

Comment:
`grp` stands for group right?
Can you be more explicitamente and use `group` instead?